tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -258,6 +258,24 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
       Seq("\u001eTEST"))
   }
 
+  test("horizonal whitespace - find") {
```

Comment:
I think now that we have the Python integration tests, we might not need these Scala tests anymore.